src/scss/_about.scss

```diff
@@ -0,0 +1,49 @@
+//---------------Break Points---------------//
+//50em (800px)
+//61.25em (980px)
+
+.about {
+  background-color: $color-blue;
+  text-align: center;
+  padding: 4em 0; //64px 0
+}
+
+.about__heading {
+  width: 10em; //160px
+  font-family: $font-egyptian;
+  font-size: 1em; //16px
+  color: $color-white;
+  border-top: 2px solid $color-white;
+  border-bottom: 2px solid $color-white;
+  padding: 0.25em 0; //4px 0
+  margin: auto;
+
+  @media(min-width: 61.25em) {
+    font-size: 1.5em; //24px
+  }
+}
+
+.about__text {
+  width: 90%;
+  font-family: $font-avenir-roman;
+  font-size: 1.25em; //20px
+  font-weight: 400;
+  color: $color-white;
+  margin: 2em auto; //32px
+
+  @media(min-width: 50em) {
+    width: 27em; //432px
+    font-size: 1.75em; //28px
+  }
+}
+
+.about__logo {
+  background-position: center;
+  display: block;
+  height: 8em; //128px
+  text-indent: -9999em;
+
+  @media(min-width: 50em) {
```

Comment:
This is the 2nd time you've used 50em as a breakpoint. Whenever you use a breakpoint more than once, it's a good idea to use a variable. That way, if you ever want to adjust your breakpoint, you only need to change it in one place.

`$bp-footer-medium: 50em;`

When creating breakpoint variables (or any variable for that matter), you need to decide whether that variables will be useful globally across all your styles or just in the content of this file/component. If you only use it in this file, then define your variable at the top of the file or, if you are nesting your component rules, you can limit the scope of the variable by putting the definition inside the first selector of your nesting structure. If 50em is used as a breakpoint in other stylesheets/components, then add a variable to your `_variables.scss` file.